### DISCUSSION TITLE
Add a TCP healthcheck to container

### DISCUSSION THREAD

**Discussion by abaschen** [mbround18/valheim-docker#282] [Ideas]
It needs a way to check that container is healthy on a TCP port. For instance, on AWS the target group need a healthcheck to allow connectivity.
A simple http 200 on port 80 is enough but a check on processes could be good too

**Comment by mbround18** [comment]
Why does it need a tcp healthcheck? Is there not a way to configure aws to not have a health check? Or some alternative work around?

**Comment by mbround18** [comment]
https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html 

It looks like all you need to do is copy the healthcheck I have inside the docker file

**Reply by abaschen** [reply]
hhaaaa nice, I didn't see the "command" for health check. thanks!
